packages/exo/test/test-exo-wobbly-point.js

```diff
@@ -0,0 +1,205 @@
+/**
+ * Based on the WobblyPoint inheritance examples in
+ * https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/google-caja/caja-spec-2007-12-21.pdf
+ * and
+ * test-far-wobbly-point.js
+ */
+
+/* eslint-disable class-methods-use-this */
+/* eslint-disable max-classes-per-file */
+/* eslint-disable-next-line import/order */
+import { test } from './prepare-test-env-ava.js';
+
+// TODO enable import of getMethodNames without deep import
```

Comment:
Agreed.  `eventual-send` is a layer atop `pass-style` so maybe best to put it there?

Reply:
Actually, pass-style devDepends on eventual-send, not the other way.

Reply:
> Actually, pass-style devDepends on eventual-send, not the other way.

Oh, I was confused. This actually makes sense: eventual-send doesn't rely on the pass-style or marshal rules.

Sorry for the mixup.